cmd/gapid/main.go

```diff
@@ -229,23 +247,36 @@ func checkVM(java string, checkVersion bool) bool {
 
 	version, err := exec.Command(java, "-version").CombinedOutput()
 	if err != nil {
+		c.logIfVerbose("Not using " + java + ": failed to get version info")
+		return false
+	}
+
+	versionStr := string(version)
+
+	// Blacklist the Google custom JDKs.
+	if p := strings.Index(versionStr, googleInfix); p >= 0 {
+		c.logIfVerbose("Not using " + java + ": is a Google JDK")
```

Comment:
I think in this case we should clearly log how to work around this issue:
"Not using Google JDK: please install openjdk version >= 8 and set its path in JAVA_HOME".

Otherwise, the average Googler user experience is: "Gapid refuses to start. With --verbose-startup, it just says it refuses to use my Google JDK".

Reply:
This is not necessary, since the .deb package causes the JDK to be installed already. This changes just causes us to ignore the environment telling us to use a different JDK than the default.

I've added a go link that gets printed in the log message now.